test/unit/accessibility/describe.js

```diff
@@ -0,0 +1,140 @@
+suite('describe', function() {
+  let myp5;
+  let myp5Container;
+  let myID = 'myCanvasID';
+  let a = 'a';
+  let b = 'b';
+  let c = 'c';
+
+  setup(function(done) {
+    myp5Container = document.createElement('div');
+    document.body.appendChild(myp5Container);
+    new p5(function(p) {
+      p.setup = function() {
+        let cnv = p.createCanvas(100, 100);
+        cnv.id(myID);
+        myp5 = p;
+        done();
+      };
+    }, myp5Container);
+  });
+
+  teardown(function() {
+    myp5.remove();
+    if (myp5Container && myp5Container.parentNode) {
+      myp5Container.parentNode.removeChild(myp5Container);
+    }
+    p5Container = null;
+  });
+
+  const getInner = elID => {
+    return document.getElementById(elID).innerHTML;
+  };
+
+  suite('p5.prototype.describe', function() {
+    let expected = 'A.';
```

Comment:
the describe tests all looks great! another good test to add would be to make sure a period isn't added if the supplied string already has one. additionally, what should the outcome be if the user supplies a string that ends in ! or ?

Reply:
I've added an exception for ! and ? in regex function